game/webcam.hh

```diff
@@ -38,6 +38,9 @@ class Webcam {
 	Dimensions const& dimensions() const { return m_texture.dimensions; }
 
   private:
+#ifdef __clang__
+	[[maybe_unused]]
+#endif
```

Comment:
Can be removed i think?

Reply:
It can't.
gcc is stupid and emits a warning about not emitting a warning in the first place, and that breaks the build due to `-Werror`

Reply:
That's a shame. Then that's alright for now. Maybe it's fixed in some later version of gcc